src/core/api.c

```diff
@@ -667,22 +663,9 @@ mender_api_publish_inventory_data(mender_keystore_t *inventory) {
     }
     cJSON_AddStringToObject(item, "name", "device_type");
     cJSON_AddStringToObject(item, "value", api_config.device_type);
```

Comment:
Just a heads-up (or a double-checking),

In the next task, the idea is that `artifact_name` and `device_type` are default inventory callbacks or built-in callbacks, that they can be disabled by the user. This is mimicking the classic client functionality, were we install a set of inventory scripts by default but can be opted out by users

Reply:
Yup, I know. And I take it this will be a persistent callback/inventory, right? Do we need to have one callback for each or can both be added by a single callback?

Reply:
Single callback, together with the rest of the device provides.